Engine/src/Engine/Audio.h

```diff
@@ -3,6 +3,8 @@
 #define _AUDIO_H_
 
 #include "irrKlang.h"
+#include <string>
```

Comment:
I don't see either of these includes being used in the header. Consider moving to .cpp